src/app/analytics/analytics.factory.js

```diff
@@ -540,6 +586,16 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
       } catch (e) {
         // Error caught in analyticsFactory.track
       }
+    },
+    trackGTM: function (eventName, data) {
```

Comment:
So far you aren't sending `data` in any of the calls to this function.  Are you planning to use it in the future?

Reply:
Yeah, good point. Any event that needs data I'll probably just make a custom analytics function for so I'll remove the data parameter.